nvcodec/platform/redist/pom.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.bytedeco</groupId>
+    <artifactId>javacpp-presets</artifactId>
+    <version>1.5.6-SNAPSHOT</version>
+    <relativePath>../../../</relativePath>
+  </parent>
+
+  <groupId>org.bytedeco</groupId>
+  <artifactId>nvcodec-platform-redist</artifactId>
+  <version>11.0.10-${project.parent.version}</version>
+  <name>JavaCPP Presets Platform Redist for NVIDIA Video Codec</name>
```

Comment:
I don't think we need a "redist" package for this. All the libraries come with the video drivers, am I correct?

Reply:
Yeah, your right! i know that encoder included in the driver. I was not sure about decoder.

i tested it and we don't need a redist.

i will remove redist packages